dao/src/main/resources/com/redhat/ipaas/dao/deployment.json

```diff
@@ -498,7 +498,8 @@
       "name": "Timed Pull to Post Example",
       "description": "This is an example of a Timed Pull to Post Integration",
       "configuration": "",
-      "desiredStatus": "Activated",
+      "desiredStatus": "Deactivated",
```

Comment:
Which was available, but there was no point in using as none of the data used it. @gashcrumb WDYT?